lang/cs/Org.Apache.REEF.Tests/Functional/Telemetry/TestMetricsMessage.cs

```diff
@@ -41,30 +41,30 @@ public void TestMetricsMessages()
             var receivedCounterMessage = GetMessageCount(lines, "Received 2 counters with context message:");
             Assert.True(receivedCounterMessage > 1);
 
-            var systemtates = GetMessageCount(lines, "TestSystemState");
-            Assert.Equal(4, systemtates);
+            var systemstates = GetMessageCount(lines, MetricsDriver.EventPrefix);
```

Comment:
Why is this variable called `systemstates`? Should be `messageCount`, right? (Also, don't forget the camelCase)

Reply:
Done